components/cards/basic-cards/basic-cards.css

```diff
@@ -0,0 +1,206 @@
+@import url(/css/base.css);
+@import url(/css/utility.css);
+@import url('https://fonts.googleapis.com/css2?family=Monoton&family=Nunito:wght@200&family=Rubik&family=Sen&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(230, 233, 236);
+}
+
+/* styling on cards components */
+.et_b-main {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    height: 500px;
+}
+
+.et_b-main>* {
+    margin: 2rem;
+}
+
+/* basic component styling*/
+.et_b-card-container {
+    background-color: rgb(235, 245, 245);
+    border-radius: 0.7rem 0.7rem 0.3rem 0.2rem;
+    box-shadow:  15px 15px 20px 3px rgb(27, 27, 27);
+    height: 300px;
+    width: 280px;
+}
+
+/* styling on all direct childs of .ed_card-container */
+.et_b-card-container>* {
+    margin-bottom: 0.5rem;
+}
+
+/* styling all the general siblings of .ed_card-subcontainer1 */
+.et_b-card-subcontainer1~* {
+    font-family: 'Monoton', cursive;
+    font-family: 'Nunito', sans-serif;
+    font-family: 'Rubik', sans-serif;
+    font-family: 'Sen', sans-serif;
+    margin-left: 0.6rem;
+    margin-right: 0.6rem;
+}
+
+.et_b-card-subcontainer1 {
+    background-image: url(https://wallpaperaccess.com/full/3846486.jpg);
+    background-size: cover;
+    border-top-left-radius: inherit;
+    border-top-right-radius: inherit;
+    width: 100%;
+    height: 50%;
+}
+
+.et_b-card-subcontainer2 {
```

Comment:
you can give naming for sub-container instead of number ( just a suggestion, please ignore this comment if not required)